content/millennium/r4/individuals/practitioner.md

```diff
@@ -89,6 +89,58 @@ List an individual Practitioner by its id:
 
 The common [errors] and [OperationOutcomes] may be returned.
 
+## Create
+
+Create an individual Practitioner.
+
+    POST /Practitioner
+
+### Authorization Types
+
+<%= authorization_types(practitioner: true, patient: false, system: true) %>
+
+### Headers
+
+<%= headers head: {Authorization: '&lt;OAuth2 Bearer Token>', 'Content-Type': 'application/fhir+json'} %>
+
+### Body Fields
+
+<%= definition_table(:practitioner, :create, :r4) %>
+
+### Example
+
+#### Request
+
+    POST https://fhir-ehr.sandboxcerner.com/r4/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/Practitioner
+
+#### Body
+
+  <%= json(:r4_practitioner_create) %>
+
+#### Response
+
+<%= headers status: 201 %>
+<pre class="terminal">
+Cache-Control: no-cache
+Content-Length: 0
+Content-Type: application/fhir+json
+Date: Fri, 6 Dec 2019 15:59:33 GMT
+Etag: W/"0"
+Last-Modified: Fri, 6 Dec 2019 15:59:30 GMT
+Location: https://fhir-ehr.sandboxcerner.com/r4/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/Practitioner/1994021
+Server-Response-Time: 3890.363996
+Status: 201 Created
+Vary: Origin
+X-Request-Id: 1638e30e497b93ff4383b2ff0eaeea68
+X-Runtime: 3.890282
+</pre>
+
+The `ETag` response header indicates the current `If-Match` version to use on subsequent updates.
```

Comment:
Do we normally document this if we don't support updates yet?

Reply:
Great catch, removed:
https://github.com/cerner/fhir.cerner.com/pull/330/commits/c128b759084993b859b499ef4e9e83aedf8a4898#diff-00a306a2aa1334ff6f799b4200c75030L138